src/app/core/api/app/auth.api.ts

```diff
@@ -41,4 +41,30 @@ export class AuthApi {
       'GET',
     );
   }
+  /**
+   * sendPasswordResetEmail
+   * @param emailValue Email cadastrado que será enviado na solicitação http
+   * @returns Retorna o resultado da verificação da existência do email do BD.
+   */
+  async sendPasswordResetEmail(emailValue: string | any): Promise<string> {
+    const response = await this.httpRequestService.sendHttpRequest(
+      '/v1/a_ser_definido',
+      'POST',
+      emailValue,
+    );
+    return response;
+  }
+  /**
+   * sendPasswordResetEmail
+   * @param emailValue Email cadastrado que será enviado na solicitação http
+   * @returns Retorna o resultado da verificação da existência do email do BD.
+   */
+  async sendNewPassword(emailValue: string | any): Promise<string> {
+    const response = await this.httpRequestService.sendHttpRequest(
+      '/v1/a_ser_definido',
```

Comment:
Coloca um comentário aqui:
//TODO: Trocar nome da rota assim que for definido.
Ou algo do tipo